test/unit/touch-enroll-specs.js

```diff
@@ -0,0 +1,169 @@
+// transpile:mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { exec as execNode } from 'child_process';
+import B from 'bluebird';
+import { getUserDefault, setUserDefault, setTouchEnrollKey, setTouchEnrollKeys, getTouchEnrollKeys, touchEnrollMenuKeys, 
+  restoreTouchEnrollShortcuts, NS_USER_KEY_EQUIVALENTS, TOUCH_ENROLL_KEY_CODE } from '../../lib/touch-enroll';
+
+const exec = B.promisify(execNode);
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('touch-enroll.js', () => {
+
+  describe('getUserDefaults(), setUserDefaults()', () => {
+    let domain = 'fakeDomain';
+
+  // Nuke the contents of the domain so it's fresh every test case
+    let clearFakeDomain = async function () {
```

Comment:
I don't know that there is any risk at all. I just don't think unit tests should be doing _anything_ real. The inputs and the outputs should be known. What if the test fails somewhere before resetting this fake domain?